server/router.ts

```diff
@@ -149,6 +150,7 @@ function setupRouter(
     next();
   });
 
+  router.post('/updateAddress', updateAddress.bind(this, models));
```

Comment:
My one remaining concern with this PR is that we fetch the `req.user.id` field from the passport/jwt verification, so this route will need to have the `passport.authenticate` binding. But -- we need to confirm that the user will actually have a jwt generated at that point in time (i.e. is logged in). Is this the case?

Reply:
add `passport.authenticate` in the route